ethereumj-core/src/main/java/org/ethereum/listener/EthereumListener.java

```diff
@@ -150,4 +160,86 @@ default void onBlock(BlockSummary blockSummary, boolean best) {
     void onTransactionExecuted(TransactionExecutionSummary summary);
 
     void onPeerAddedToSyncPool(Channel peer);
+
+    EthereumListener STUB = new EthereumListener() {
```

Comment:
I don't think this either necessary. Let it use old deprecated `EthereumListenerAdapter`

Reply:
Ok, I'll replace it with `EthereumListenerAdapter`